src/webapp/reports/glass-admin/glass-admin-list/atc-classification/ATCClassificationList.tsx

```diff
@@ -0,0 +1,192 @@
+            .sortBy(column => indexes[column.name] || 0)
+            .value();
+    }, [tableProps.columns, visibleColumns]);
+
+    return (
+        <React.Fragment>
+            <StyledButtonContainer>
+                <Button onClick={openUploadATCModal} color="primary" variant="contained">
+                    {i18n.t("Upload new ATC file")}
+                </Button>
+
+                {!isRecalculated ? (
+                    <Button onClick={openRecalculateLogicModal} color="primary" variant="contained">
+                        {i18n.t("Recalculate logic")}
+                    </Button>
+                ) : (
+                    <>
+                        <Button color="default" variant="contained" disabled>
+                            {i18n.t("Recalculation Ongoing...")}
+                        </Button>{" "}
+                        <CancelButton onClick={cancelRecalculation} variant="contained">
+                            {i18n.t("Cancel recalculation")}
+                        </CancelButton>
+                    </>
+                )}
+            </StyledButtonContainer>
+
+            <ObjectsList<ATCViewModel>
+                {...tableProps}
+                columns={columnsToShow}
+                onChangeSearch={undefined}
+                onReorderColumns={saveReorderedColumns}
+            />
+
+            <GLASSAdminDialog
+                selectedItems={selectedItems}
+                isOpen={isPatchModalOpen}
+                closeModal={closePatchModal}
+                description={
+                    isCurrentVersion
+                        ? "You are replacing the latest ATC file. It will become the default version"
+                        : "You are replacing an old ATC file. It won't affect calculations"
+                }
+                title="Patch version"
+                saveFile={patchVersion}
+            />
+
+            <GLASSAdminDialog
+                isOpen={isUploadATCModalOpen}
+                closeModal={closeUploadATCModal}
+                description="All years will be overwritten with the data provided in this file."
+                title="Upload new ATC file"
+                uploadedYears={uploadedYears}
+                saveFile={uploadATCFile}
+            />
+
+            <ConfirmationDialog
+                isOpen={isRecalculateLogicModalOpen}
+                title={"Recalculate"}
+                onSave={saveRecalculationLogic}
+                onCancel={closeRecalculateLogicModal}
+                saveText={i18n.t(isRecalculating ? "Recalculating..." : "Start recalculation")}
+                cancelText={i18n.t("Cancel")}
+                disableSave={isRecalculating}
+                maxWidth="md"
+                fullWidth
+            >
+                {previousVersionExists && (
+                    <p>Last recalculation was done with {getVersion(tableProps.rows, "previousVersion")}</p>
```

Comment:
Use translations in this lines. See example:
i18n.t("Last recalculation was done with  {{previousVersion}}", { previousVersion: getVersion(tableProps.rows, "previousVersion") })